install-pip-dependencies.sh

```diff
@@ -41,7 +41,7 @@ fi
 # Install the packages from the requirements file
 export MPI4PY_BUILD_CONFIGURE=1
 
-if [[ $(hostname) == tioga* ]]; then
+if [[ $(hostname) == tioga* || $(hostname) == odyssey || $(hostname) == tuolumne* ]]; then
     # Need extra flags for the mpi4py build
     LDFLAGS="$LDFLAGS -Wl,-rpath $CRAYLIBS_X86_64" \
```

Comment:
Sorry for being nosy here: Is it really the case that without this flag, MPI silently produces wrong results? (That's the sense I got from the meeting this morning.)

Reply:
Thanks for your question! No, this fixes startup errors like `ImportError: libmodules.so.1: cannot open shared object file: No such file or directory`. It is not related to the wrong results we discussed in the morning.

Reply:
Unfortunately not :(